### PR TITLE
perf(lodash): avoid importing global lodash

### PR DESCRIPTION
**Pull request by jleveugle**
Avoid importing global lodash and get these kind of errors: 

```
@ovh-ux/manager-freefax: [BABEL] Note: The code generator has deoptimised the styling of /Users/jleveugle/Documents/Work/github/ovh-ux/manager/node_modules/lodash/lodash.js as it exceeds the max of 500KB.
```

Smaller is better :D

### CODE REVIEW

File: packages/manager/overthebox/src/order/order-overTheBox.controller.js [hunk 1]
```diff
@@ -1,4 +1,4 @@
-import _ from 'lodash';
+import first from 'lodash/first';
```
Comment (FredericEspiau) [inline packages/manager/overthebox/src/order/order-overTheBox.controller.js]: Please use `head` instead

https://github.com/lodash/lodash/wiki/Changelog#compatibility-warnings

File: yarn.lock [hunk 2]
```diff
@@ -1866,11 +1866,6 @@ ajv@^6.5.3, ajv@^6.5.5:
     json-schema-traverse "^0.4.1"
     uri-js "^4.2.2"
 
-amdefine@>=0.0.4:
```
Comment (FredericEspiau) [inline yarn.lock]: Lots of changes not related to the PR

File: packages/manager/overthebox/src/details/overTheBox-details.constant.js [hunk 3]
```diff
@@ -1,4 +1,5 @@
-import _ from 'lodash';
+import first from 'lodash/first';
```
Comment (FredericEspiau) [inline packages/manager/overthebox/src/details/overTheBox-details.constant.js]: Please use `head` instead

https://github.com/lodash/lodash/wiki/Changelog#compatibility-warnings

File: packages/manager/freefax/src/notifications/freefax-notifications.factory.js [hunk 4]
```diff
@@ -1,5 +1,10 @@
 import angular from 'angular';
-import _ from 'lodash';
+
+import extend from 'lodash/extend';
```
Comment (FredericEspiau) [inline packages/manager/freefax/src/notifications/freefax-notifications.factory.js]: Please use `assignIn` instead

https://github.com/lodash/lodash/wiki/Changelog#compatibility-warnings

File: yarn.lock (hunk 2 above)
Comment (jleveugle) [reply yarn.lock]: due to yarn upgrade :)

**Comment by antleblanc** [comment]
According to the `warning?` message you have from `manager-freefax` section, what's about applying the `perf` commit to this specific section instead?

It is a bit confusing here if I pay attention to the whole diff because you are targeting multiple `workspace` such as `core` and `overthebox` for instance.

I'm not convince about specify `lodash` as a `scope` to your commit message but it's probably a personal feeling.

What is your point of vue for example targeting a `section|workspace` and then applying the related `perf` commit to?

File: packages/manager/overthebox/src/details/overTheBox-details.constant.js (hunk 3 above)
Comment (jleveugle) [reply packages/manager/overthebox/src/details/overTheBox-details.constant.js]: done :)

File: packages/manager/overthebox/src/order/order-overTheBox.controller.js (hunk 1 above)
Comment (jleveugle) [reply packages/manager/overthebox/src/order/order-overTheBox.controller.js]: done

File: packages/manager/freefax/src/notifications/freefax-notifications.factory.js (hunk 4 above)
Comment (jleveugle) [reply packages/manager/freefax/src/notifications/freefax-notifications.factory.js]: done

@FredericEspiau thx for reviewing ;)

**Comment by jleveugle** [comment]
@antleblanc The perf commit is cross workspaces, each workspace will get a trace in the history / changelog with specific files for the current workspace :)

We must see here a way to standardize the changes across all the workspaces (and the message commit too :)) 

For the scope, I tried. I can update it if needed, what is your purpose? :)

**Comment by antleblanc** [comment]
Due to the fact that you keep only a single commit, the scope `lodash` sounds good.

The thing that I'm pointing out here is the description which is related to `@ovh-ux/manager-freefax`. 

This warning will be also displayed with the `core` and `overthebox` package too.

**Pull request merged**